kod/object/item/passitem/attmod/frozjewl.kod

```diff
@@ -122,30 +131,28 @@ messages:
 
    SendAnimation()
    {
-      AddPacket(1,ANIMATE_NONE,2,2);
-
-      return;
-   }
+      % Override to replace the default broken inventory graphic `viBroken_group` 
+      % with a broken ground graphic.
+      if piHits <= 0
+      {
+         AddPacket(1,ANIMATE_NONE, 2,FRAME_BROKEN_GROUND);
 
-   SendLookAnimation()
-   {
-      AddPacket(1,ANIMATE_NONE,2,1);
+         return;
+      }
 
-      return;
-   }
+      propagate;
+   } 
 
    SendInventoryAnimation()
    {
       if piAnimation = ANIM_FLASH
       {
-         AddPacket(1,ANIMATE_ONCE,4,300,2,3,2,3,2,1);
-      }
-      else
-      {
-         AddPacket(1,ANIMATE_NONE,2,1);
+         AddPacket(1,ANIMATE_ONCE, 4,ANIMATION_FLASH_SPEED, 2,FRAME_FLASH, 2,FRAME_FLASH, 2,FRAME_DEFAULT);
```

Comment:
Just in case it's unclear what's going on here: If you have a JOF equipped and "enable" a spell like touch of flame, your spell attacks will animate the inventory object to show this "flash" state.

![image](https://github.com/user-attachments/assets/dc835084-ee64-4865-9cba-1566ea8ba8d3)